server/tests-py/queries/graphql_query/basic/select_query_batching_with_mutation.yaml

```diff
@@ -0,0 +1,42 @@
+description: GraphQL query to test batching with mixed queries and mutations
+url: /v1/graphql
+status: 200
+response:
+  - data:
+      update_user:
+        affected_rows: 1
+        returning:
+        - id: '2'
+  - data:
+      user:
+      - id: '1'
+        name: User 1
+      - id: '2'
+        name: Temp User
+  - data:
+      update_user:
+        affected_rows: 1
+        returning:
+        - id: '2'
+query:
+  - query: |
+      mutation {
```

Comment:
I'm not sure if this is poor form (updating the test data set from a test like this), so I try to put back the original data at the end so that it doesn't affect any other tests.

Reply:
I think the test setup/teardown actions run around each individual test, not around the whole test group, so I *think* this is unnecessary.